rpm/exec_phonesim

```diff
@@ -1,14 +1,18 @@
-#!/bin/bash
-if [ "$1" == "1" ]; then
+#!/bin/sh
+set_phonesim_property() {
+    /usr/sbin/run-blts-root /usr/bin/dbus-send --system --print-reply --type=method_call --dest=org.ofono /phonesim org.ofono.Modem.SetProperty string:"$1" variant:boolean:"$2" > /dev/null
```

Comment:
An alternative to run-blts-root would be to relax ofono access checks, but I'd need to check how to do that. I'm not sure which of these is the better option.

Reply:
At least this is nicely contained in phonesim and regardless of blts-tools this function makes the script cleaner.

Reply:
I might be worth it to split each argument into its own line for formatting reasons.

Reply:
It looks like ofono supports only one configuration file for relaxing those checks. I could have it as part of the tests though, I think.

Reply:
Okay, now this runs as privileged instead of root. Seems to work.

> I might be worth it to split each argument into its own line for formatting reasons.

Improved this as well. (It's not quite each argument on its own line, though.)